src/py/reactpy/reactpy/backend/utils.py

```diff
@@ -8,62 +8,51 @@
 from importlib import import_module
 from typing import Any
 
-from reactpy.backend.types import BackendImplementation
+from reactpy.backend.types import BackendProtocol
 from reactpy.types import RootComponentConstructor
 
 logger = logging.getLogger(__name__)
 
-SUPPORTED_PACKAGES = (
-    "starlette",
+SUPPORTED_BACKENDS = (
     "fastapi",
     "sanic",
     "tornado",
     "flask",
+    "starlette",
 )
 
 
 def run(
     component: RootComponentConstructor,
-    host: str = "127.0.0.1",
+    host: str | None = None,
     port: int | None = None,
-    implementation: BackendImplementation[Any] | None = None,
+    implementation: BackendProtocol[Any] | None = None,
 ) -> None:
     """Run a component with a development server"""
     logger.warning(_DEVELOPMENT_RUN_FUNC_WARNING)
 
     implementation = implementation or import_module("reactpy.backend.default")
-
     app = implementation.create_development_app()
     implementation.configure(app, component)
-
-    host = host
+    host = host or "127.0.0.1"
     port = port or find_available_port(host)
-
     app_cls = type(app)
+
     logger.info(
-        f"Running with {app_cls.__module__}.{app_cls.__name__} at http://{host}:{port}"
+        "ReactPy is running with '%s.%s' at http://%s:%s",
+        app_cls.__module__,
+        app_cls.__name__,
+        host,
+        port,
```

Comment:
There's probably a bunch of other places where I used f-string in logs

Reply:
Yup, I didn't want to clutter this PR too much though. I think covering other files should be a follow up PR.